ding-dong.cabal

```diff
@@ -72,7 +72,6 @@ library
                       , scientific
     hs-source-dirs:   src
     default-language: Haskell2010
-    extra-lib-dirs:/home/branko/Documents/building-nostr-wasm/secp-wasm-compiled
```

Comment:
We really don't need it, but we need to export the location for pkg-config, e.g. `export PKG_CONFIG_PATH=some_directory/lib/pkgconfig`. Maybe the nix section also needs to be adjusted for that, but I'm not touching that part.